### PR TITLE
CONTRIBUTING.md: Added PMM API documentation info

### PR DESCRIPTION
**Pull request by LenzGr**
Added section on how to contribute to the PMM API documentation.

Signed-off-by: Lenz Grimmer <lenz.grimmer@percona.com>

PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,9 +58,37 @@ This project is built from several repositories:
 * [percona/pmm-qa](https://github.com/percona/pmm-qa)
 * [percona/pmm-doc](https://github.com/percona/pmm-doc)
 
-## API documentation
+## API Reference Documentation
 
-See API definitions [here](https://percona-pmm.readme.io/reference/introduction).
+You can review the PMM API definition [here](https://percona-pmm.readme.io/).
+
+The API documentation source files are located in the `markdown` directory. The
```
Comment (BupycHuk) [inline CONTRIBUTING.md:65]: I think we also should mention that API documentation is generated from the swagger.json file.

Comment (LenzGr) [reply CONTRIBUTING.md:65]: Good idea, I'll add that.

Comment (LenzGr) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,9 +58,39 @@ This project is built from several repositories:
 * [percona/pmm-qa](https://github.com/percona/pmm-qa)
 * [percona/pmm-doc](https://github.com/percona/pmm-doc)
 
-## API documentation
+## API Reference Documentation
 
-See API definitions [here](https://percona-pmm.readme.io/reference/introduction).
+You can review the PMM API definition [here](https://percona-pmm.readme.io/).
+
+It is generated using the file `api/swagger/swagger.json` and additional API
```
Comment (atymchuk) [inline CONTRIBUTING.md:65]:
```suggestion
It is generated from our `.proto` [files](./api/) using a special [OpenAPI v2 tool](https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-openapiv2) and additional API
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Comment by LenzGr** [comment]
Is the PR OK to be merged like this? Or should I squash and rebase the changes to the `CONTRIBUTING.md` file into a single commit before?

**Review by BupycHuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**